conversions.py

```diff
@@ -5,4 +5,10 @@ def miles_to_km(num):
     return float(num)*1.60934
 
 def in_to_cm(num):
-    return num*2.54
\ No newline at end of file
+    return num*2.54
+
+def miles_to_yards(num):
+    return num*176
+
+def yards_to_feet(num):
+    return num*30
```

Comment:
```suggestion
    return num*3
```
Looks like I've found the missing factor of 10 from above. Great work.